### PR TITLE
Update setup.cfg

### LINKED ISSUE #1: Specify minimum versions of Python and Django
We need to specify that the minimum supported version for Python is `3.6` (I saw it uses type hints `List`) and about Django I am not sure.

### PR DESCRIPTION
**Pull request by DmytroLitvinov**
Closes #1

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -10,16 +10,28 @@ author_email = guettliml.django-check-html-middleware@thomas-guettler.de
...
+    Framework :: Django :: 3.2
+    Framework :: Django :: 4.0
     Intended Audience :: Developers
     License :: OSI Approved :: MIT License
     Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
```
Comment (guettli) [inline setup.cfg]: Is this list needed? I guess nobody uses a 3.5 or lower. And nevertheless some lines later there is >=3.6.

The rest looks good.  I will merge it after we agreed on above topic. Thank you.

**Review by guettli — CHANGES_REQUESTED** [review]

File: setup.cfg (hunk 1 above)
Comment (DmytroLitvinov) [reply setup.cfg]: It will be shown at PyPI. 
Here is an [example how it looks at PyPI](https://pypi.org/project/djangorestframework/)

Comment (guettli) [reply setup.cfg]: Who will update it, when Python 3.11 will be released? I am super lazy. I don't want to do it. Please tell me why the explicit versions are needed. If they are not needed, then please update the PR.

Comment (DmytroLitvinov) [inline setup.cfg]:
```suggestion
```

**Comment by DmytroLitvinov** [comment]
@guettli , removed classifiers for Python

**Review by guettli — APPROVED** [review]
Looks good to me.

**Comment by guettli** [comment]
@DmytroLitvinov I merged your PR. Thank you!

**Pull request merged**